Cargo.toml

```diff
@@ -17,12 +17,12 @@ members = []
 rustc_version = "0.1.7"
 
 [dependencies]
-chrono = "0.3"
-error-chain = "0.8.1"
+chrono = "0.4"
+error-chain = { git = "https://github.com/rnewman/error-chain", branch = "rnewman/sync" }
```

Comment:
Are the related changes going to be pushed back to `rust-lang-nursery/error-chain`? I wouldn't want to be maintaining a fork for too long.

Reply:
Not directly. They added `Sync` in 0.8.1, then they removed it in 0.9, with figuring out how to allow consumers to opt in to different kinds of errors being a bit TBD. This is a very simple fork, at least.